### PR TITLE
DEVEX-2305 Add dxpy dependencies cross-platform testing suite 

### PR DESCRIPTION
**Pull request by wormsik**

### CODE REVIEW

File: build/dependencies_sanity_tests/.gitignore
[the remark this answers is not in the document]
Comment (wormsik) [reply build/dependencies_sanity_tests/.gitignore:1]: I am ok with any of them...

@kpjensen what do you think?

[the remark this answers is not in the document]
Comment (kpjensen) [reply build/dependencies_sanity_tests/.gitignore:1]: What about `dependencies_cross_platform_tests`?

**Review by kpjensen — APPROVED** [review]

File: build/dependencies_sanity_tests/.gitignore
[the remark this answers is not in the document]
Comment (commandlinegirl) [reply build/dependencies_sanity_tests/.gitignore:1]: Sg!

File: build/dependencies_sanity_tests/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+### colorama
+
+Colorized terminal output on Windows. Called directly in `dx` and `dx-app-wizard` scripts. Don't know how to test automatically.
+
+### cryptography
+
+Seems to be unused and present for historical reasons. In dxpy, the library is NOT used anywhere. 
+
+The only place I could found is `urllib3` contrib module `PyOpenSSL` (https://urllib3.readthedocs.io/en/stable/reference/contrib/pyopenssl.html) and it is required for certificate validation (using packace `urllib3[secure]`) in Python 2: https://urllib3.readthedocs.io/en/stable/user-guide.html#certificate-verification-in-python-2 and https://urllib3.readthedocs.io/en/stable/user-guide.html#certificate-verification.
```
Comment (commandlinegirl) [inline build/dependencies_sanity_tests/README.md:38]:
```suggestion
The only place where it seems to be used is with the `urllib3` contrib module `PyOpenSSL` (https://urllib3.readthedocs.io/en/stable/reference/contrib/pyopenssl.html) and it is required for certificate validation (using packace `urllib3[secure]`) in Python 2: https://urllib3.readthedocs.io/en/stable/user-guide.html#certificate-verification-in-python-2 and https://urllib3.readthedocs.io/en/stable/user-guide.html#certificate-verification.
```

File: build/dependencies_sanity_tests/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# dxpy dependencies sanity tests
```
Comment (commandlinegirl) [inline build/dependencies_sanity_tests/README.md:1]:
```suggestion
# dxpy dependencies cross platform tests
```

**Review by commandlinegirl — APPROVED** [review]
LGTM modulo minor comments

**Review by commandlinegirl — APPROVED** [review]
Very nice work, thanks!

**Pull request merged**